src/message/buttons/urlButton.js

```diff
@@ -35,7 +35,13 @@ export default class UrlButton extends Button {
         return this;
     }
 
+    setMessengerExtensions(value = false) {
```

Comment:
can we switch this to `useMessengerExtensions` with use `true` as the default value